scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -23,7 +23,6 @@ sh $INSTALL_DOCKER
 
 export PATH="$PATH:$(pwd)/bin"
 export ARTIFACTS_DIR="/tmp/artifacts"
```

Comment:
Remove this this line too. `ARTIFACTS_DIR` is a CI defined variable. Do not overwrite the default artifacts directory  unless you are very sure.